tests/test_io.py

```diff
@@ -509,7 +509,14 @@ def test_read_all_variables(self):
         for var in res.name:
             res.get_variable_data(var)
     
-    
+    @testattr(stddist = True)
+    def test_read_all_variables_from_stream(self):
```

Comment:
This test does not ensure that res.name actually contains any names and that get_variable_data returns something meaningful. One option is to compare loading without a stream.

Reply:
Fixed.